src/site/content/en/blog/responsive-web-design-basics/index.md

```diff
@@ -0,0 +1,582 @@
+A meta viewport tag gives the browser instructions on how to control the page's dimensions and scaling.
+
+To attempt to provide the best experience, mobile browsers render
+the page at a desktop screen width (usually about `980px`, though this varies
+across devices), and then try to make the content look better by increasing
+font sizes and scaling the content to fit the screen.
+This means that font sizes may appear inconsistent to users,
+who may have to double-tap or pinch-to-zoom
+in order to see and interact with the content.
+
+```html/4
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    …
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    …
+  </head>
+  …
+
+```
+
+Using the meta viewport value `width=device-width` instructs the page to match
+the screen's width in device-independent pixels. A device (or density) independent pixel being a representation of a single pixel, which may on a high density screen consist of many physical pixels. This allows the page to reflow
+content to match different screen sizes, whether rendered on a small mobile
+phone or a large desktop monitor.
+
+<figure class="w-figure">
+  <img src="./without-vp-meta.png" width="500" alt="Screenshot of a page with the text hard to read as it is very zoomed out">
+  <figcaption class="w-figcaption">
+    An example of how the page loads in a device without the viewport meta tag. <a href="https://without-vp-meta.glitch.me/">See this example on Glitch</a>.
+  </figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="./with-vp-meta.png" width="500" alt="Screenshot of the same page with the text at a size that can be read">
+  <figcaption class="w-figcaption">
+    An example of how the page loads in a device with the viewport meta tag. <a href="https://with-vp-meta.glitch.me/">See this example on Glitch</a>.
+  </figcaption>
+</figure>
+
+[Some browsers](https://css-tricks.com/probably-use-initial-scale1/)
+keep the page's width constant when rotating to landscape
+mode, and zoom rather than reflow to fill the screen. Adding the value
+`initial-scale=1` instructs browsers to establish a 1:1 relationship between CSS
+pixels and device-independent pixels regardless of device orientation, and
+allows the page to take advantage of the full landscape width.
+
+{% Aside 'caution' %}
+  To ensure that older browsers can properly parse the attributes,
+  use a comma to separate attributes.
+{% endAside %}
+
+The [Does not have a `<meta name="viewport">` tag with `width` or `initial-scale`](/viewport/)
+Lighthouse audit can help you automate the process of making sure that your HTML documents are using the viewport meta tag correctly.
+
+### Ensure an accessible viewport {: #accessible-viewport }
+
+In addition to setting an `initial-scale`,
+you can also set the following attributes on the viewport:
+
+{% Compare 'worse' %}
+  * `minimum-scale`
+  * `maximum-scale`
+  * `user-scalable`
+{% endCompare %}
```

Comment:
```suggestion
* `minimum-scale`
* `maximum-scale`
* `user-scalable`
```

My suggestion isn't displaying correctly so it's probably easiest to just change it back to your original.

![image](https://user-images.githubusercontent.com/4713486/82338382-af6a8700-99a1-11ea-9300-8033c26177b0.png)